packages/cascara/src/modules/DataImage/DataImage.doc.mdx

```diff
@@ -15,15 +16,13 @@ Feel free to play around with the following code:
 <ModuleSandbox
   isEditing={true}
   record={{
-    id: 1,
-    firstName: 'John',
-    lastName: 'Doe',
+    logo: 'https://cascara.design/cascara_meta.svg',
   }}
 >
   <DataImage
-    attribute='firstName'
-    alt='My Text'
-    src='/media/examples/login-button.png'
+    attribute='logo'
+    alt='Cascara'
+    value='https://cascara.design/cascara_meta.svg'
```

Comment:
@brianespinosa this is one of the issues we saw and fixed, as they all show up in Docs.